python/lsst/afw/cameraGeom/utils.py

```diff
@@ -204,11 +238,18 @@ def makeImageFromAmp(amp, imValue=None, imageFactory=afwImage.ImageU, markSize=1
 
 
 def calcRawCcdBBox(ccd):
-    """!Calculate the raw ccd bounding box
-
-    @param[in] ccd  Detector for with to calculate the un-trimmed bounding box
-    @return Box2I of the un-trimmed Detector,
-            or None if there is not enough information to calculate raw BBox
+    """Calculate the raw ccd bounding box.
+
+    Parameters
+    ----------
+    ccd : `lsst.afw.cameraGeom.Detector`
+        Detector for with to calculate the un-trimmed bounding box
```

Comment:
"Detector for with to"

Reply:
Looks like @SimonKrughoff 's to blame for that one...